### DISCUSSION TITLE
Function Keys not working

### DISCUSSION THREAD

**Discussion by jonsmith1234** [freeplane/freeplane#1766] [Help] [answered]
I recently downloaded Freeplane (long time user of FreeMind) and like it much better.  I've found some excellent videos on YouTube by Ken Hill, but many commands he references do not work.  The default INS for a new node is a Tab and only the method of Insert, New node, New child node, then Ctrl-click, followed by a INS allowed me to change.  None of the other referenced methods in the Freeplane documentation worked.   For Function Keys, I can turn on the Function Key menu, and those options work, but that is just for the first 10 functions keys.  Something like Ctrl-F1 does not work and I cannot see any way to add a Command Search to one of the unused function keys.  I've tried reinstalling Freeplane (I loaded the version linked from today).  Assign Hot Key does not work.  There are no Preset Hot Keys nor none to load.  I'm running the latest version of Chrome on Windows 11, so all of my drivers should be up to date. 

Can anyone help me get Freeplane working as advertised?  Seems like a wonderful tool

**Comment by euu2021** [comment]
> The default INS for a new node is a Tab

The default was changed to Tab, because it has superior compatibility with different keyboard layouts.

> only the method of Insert, New node, New child node, then Ctrl-click, followed by a INS allowed me to change

Another method is to search for the command in Command Search, and Ctrl-click it.

> None of the other referenced methods in the Freeplane documentation worked.

What happens exactly? Does any error message show up?

>  Something like Ctrl-F1 does not work

Do you mean: there is a command assigned to it, but nothing happens when the hotkey is used?

> and I cannot see any way to add a Command Search to one of the unused function keys

Try to search for the command in Command Search, and Ctrl-click it. I tested, and it's working here (in the example, I assign it to Ctrl+Shift+F7):

![java_CvIJow2Hv1](https://github.com/freeplane/freeplane/assets/77707706/99d87d44-b07a-4345-b128-766c479c0df3)

**Reply by jonsmith1234** [reply]
Nothing happens when I press any of the the Function Keys.  I can activate the F-Bar under View, which has Functions 1-3 on it.  I can click with the mouse on F1, F2, or F3 on the F-bar and they work properly.   F5-F12 are not assigned and I cannot assign them.   I cannot find a way to get to Command Search.   Function keys work properly in other applications, so it isn't the keyboard.

**Reply by euu2021** [reply]
I see. So, there is a command assigned, for example, to F1, and it works if you click it with the mouse in the F-Bar, but nothing happens when you press the actual F1 key. Now, that's strange. I don't know what causes this.

> I cannot find a way to get to Command Search

It is inside Help->Command Search

**Reply by jonsmith1234** [reply]
Yes, F1 defaults to Documentation.  If I click on F1 on the F-Bar, I get the documentation.   But nothing happens when I press F1.

**Reply by euu2021** [reply]
Now, I vaguely remember having a problem like this is the past. Let's see if someone knows more about it.

**Comment by Illegit** [comment, marked as answer]
It sounds strange, but maybe I can help only if the problem is due to some preliminary topics, just in case:
1) alt+shift: have you checked the keyboard settings --> 
![image](https://github.com/freeplane/freeplane/assets/62674157/c5481683-ab64-4e91-afa6-7c25cd428884) if so select you keyboard settings
2) fn+esc: if you have a wireless keyboard be sure you inadvertently have pushed this combination, then do it again to back to your F-keys settings.
Most probable 1). If the problem is related to FP, I cannot help.

**Reply by jonsmith1234** [reply]
yes, that solved it (the Fn-Esc)!  Thanks
